.github/workflows/e2e.yml

```diff
@@ -160,19 +160,22 @@ jobs:
         working-directory: opencollective-frontend
         run: CYPRESS_INSTALL_BINARY=0 npm ci --prefer-offline --no-audit
 
+      - name: Set commit hash (frontend)
+        working-directory: opencollective-frontend
+        run: echo "FRONTEND_COMMIT_HASH=$(git rev-parse HEAD)" >> $GITHUB_ENV
+
       - name: Restore build (frontend)
         uses: actions/cache@v1
         id: next-build
         with:
           path: opencollective-frontend/dist
-          key: ${{ runner.os }}-next-build-${{ hashFiles('opencollective-frontend/**') }}
+          key: ${{ runner.os }}-next-build-${{ env.FRONTEND_COMMIT_HASH }}
 
       - name: Restore .next cache (frontend)
-        if: steps.next-build.outputs.cache-hit != 'true'
         uses: actions/cache@v1
         with:
-          path: opencollective-frontend/.next/cache
-          key: ${{ runner.os }}-next-cache-${{ hashFiles('opencollective-frontend/**') }}
+          path: opencollective-frontend/.next
+          key: ${{ runner.os }}-nextjs-${{ env.FRONTEND_COMMIT_HASH }}
```

Comment:
If you modify the `key`, you should modify the `restore-keys` similarly.

Reply:
@znarf : Sorry, this is a mistake on my part. I have readjusted this to match the key. 👍🏽